src/index.html

```diff
@@ -221,24 +206,24 @@ <h3><a class="anchor" id="matches"></a>Matches</h3>
         <li>
           At the end of a game, league points will be awarded as follows:
           <ul>
-            <li>The team with the most game points will be awarded 8 points towards the competition league.
-            <li>The team with the second most will be awarded 6.
+            <li>The team with the most game points will be awarded 4 points towards the competition league.
+            <li>The team with the second most will be awarded 2.
             <li>Teams whose robot was not entered into the round, or who were disqualified from the round, will be awarded no points.
           </ul>
         <li>
           If a match ends in a tie, robots will be awarded the average of the points that their combined
-          positions would be awarded, in this case 7 (since this is (8 + 6) &div; 2).
+          positions would be awarded, in this case 3 (since this is (4 + 2) &div; 2).
```

Comment:
With 2 robots it's much easier to say "in the event of a tie both teams get 3 points"